### PR TITLE
Feature job section#13

### PR DESCRIPTION
**Pull request by EliusCaleb**
The task to design :
A feature that is a stand-alone component. This page should be different from the element.

The text "open opportunities" is an anchor link that points to the /jobs page

To achieve this, we needed to:
    1) We need to create a standalone component  that is called job-section
    2) Add view to the components  which is HTML code
    3)Add style to the component
    4) Render this component to the app component 

## Type of change
Delete my package.lock json

# Screenshot (optional)
image

# How Has This Been Tested?
When I click the open or opportunities it redirects to /jobs

# Checklist:
- [ 1] My code follows the style guidelines of this project
- [2 ] I have performed a self-review of my code
- [3 ] I have made corresponding changes to the documentation
- [4] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: apps/elewa-group-website/src/app/app.component.html [hunk 1]
```diff
@@ -1 +1,4 @@
-<elewa-group-home-page></elewa-group-home-page>
\ No newline at end of file
+
+
+ <elewa-group-job-section></elewa-group-job-section
```
Comment (IanOdhiambo9) [inline apps/elewa-group-website/src/app/app.component.html:3]: Job section needs to be on the home page.
only home page tag should be on thing file..

File: libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.ts [hunk 2]
```diff
@@ -5,4 +5,4 @@ import { Component } from '@angular/core';
   templateUrl: './home-page.component.html',
   styleUrls: ['./home-page.component.scss'],
 })
-export class HomePageComponent {}
+export class HomePageComponent {}
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.ts:8]: This entire component needs to be inside a components folder on this lib..

Sync your fork, the components folder was already created.

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
You've also pushed your package.json files.

This cannot be merged.

File: apps/elewa-group-website/src/main.ts [hunk 3]
```diff
@@ -2,7 +2,10 @@ import 'zone.js';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 
 import { AppModule } from './app/app.module';
+// import { bootstrapApplication } from '@angular/platform-browser';
+// import { AppComponent } from './app/app.component';
 
 platformBrowserDynamic()
   .bootstrapModule(AppModule)
   .catch((err) => console.error(err));
+// bootstrapApplication(AppComponent)
```
Comment (IanOdhiambo9) [inline apps/elewa-group-website/src/main.ts:11]: No need to change this file, keep as it was

File: apps/elewa-group-website/src/app/app.module.ts [hunk 4]
```diff
@@ -1,21 +1,24 @@
...
 @NgModule({
   declarations: [AppComponent],
   imports: [
     BrowserModule, 
     ScullyLibModule,
     HomePageModule,
+    JobSectionComponent,
     CardsModule
+
   ],
```
Comment (IanOdhiambo9) [inline apps/elewa-group-website/src/app/app.module.ts:22]: No need to change this file.

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]

**Comment by EliusCaleb** [comment]
I have change the  job-section

**Comment by IanOdhiambo9** [comment]
Hello
I'm closing this pull request to to high number of conflicts not resolved.

**Comment by IanOdhiambo9** [comment]
Hello
I'm closing this pull request to to high number of conflicts not resolved and you've deleted the package.json and lock files. This will cause deletion on main branch as well. Please do a new PR without this files.

**Pull request closed without merging**